libs/ui/src/modals/screens/tokens-selector/tokens-selector.tsx

```diff
@@ -33,27 +34,33 @@ const keyExtractor = ({ tokenAddress, tokenId }: TokenType) => getTokenSlug(toke
 export const TokensSelector: FC = () => {
   const allTokensMarketInfoSelector = useTokensMarketInfoSelector();
   const {
-    params: { token }
-  } = useRoute<RouteProp<ScreensParamList, ScreensEnum.SendTokensSelector>>();
+    params: { token, field, withBalanceTokens }
+  } = useRoute<RouteProp<ScreensParamList, ScreensEnum.TokensSelector>>();
   const { navigate } = useNavigation();
   const { chainId } = useSelectedNetworkSelector();
   const visibleAccountTokens = useVisibleAccountTokensSelector();
-  const accountTokensWithBalance = useMemo(() => getTokensWithBalance(visibleAccountTokens), [visibleAccountTokens]);
+  const accountTokens = useMemo(
+    () => (withBalanceTokens ? getTokensWithBalance(visibleAccountTokens) : visibleAccountTokens),
+    [visibleAccountTokens, field]
```

Comment:
why do you have `field` in deps which is not used, but don't have `withBalanceTokens` which is used